dotCMS/src/main/java/com/dotmarketing/quartz/job/DistReindexJournalCleanupThread.java

```diff
@@ -82,9 +83,6 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
 			} catch (DotHibernateException e) {
 				Logger.error(this, e.getMessage(), e);
 			}
-			finally {

```

Comment:
Remove the entire wrapper `finally` block, because this is already happening inside the `run`